framework/src/play/src/main/scala/play/core/parsers/Multipart.scala

```diff
@@ -211,6 +211,7 @@ object Multipart {
             case key => (key.trim, "")
           }(breakOut): Map[String, String])
         _ <- values.get("form-data")
+        _ <- Option(values.keySet.contains("filename")).filter(_ == false)
```

Comment:
```suggestion
        _ <- Option(values.contains("filename")).filter(_ == false)
```

`Map.contains` test if the map contains a value for a key.